src/auth/auth.service.ts

```diff
@@ -1,14 +1,14 @@
 import { Injectable } from '@nestjs/common';
 import { sign } from 'jsonwebtoken';
-import { Payload } from 'src/schemas/users.schema';
 import { UserService } from 'src/users/users.service';
+import { Payload } from '../users/users.schema';
 
 @Injectable()
 export class AuthService {
   constructor(private userService: UserService) {}
 
   async signPayload(payload: Payload) {
-    return sign(payload, process.env.SECRET_KEY, { expiresIn: '7d' });
+    return sign(payload, process.env.JWT_SECRET_KEY, { expiresIn: '2d' });
```

Comment:
Use the configservice to achieve this instead of process.env

Reply:
okay boss